File: src/main/java/com/example/readinglist/ReadingListApplication.java
package com.example.readinglist;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

// enable component-scanning and auto-configuration
@SpringBootApplication
public class ReadingListApplication {

public static void main(String[] args) {
    // bootstrap application
    SpringApplication.run(ReadingListApplication.class, args);
  }
}

File: src/main/kotlin/readinglist/ReadingListApplication.kt
package readinglist;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

@SpringBootApplication
class ReadingListApplication

fun main(args: Array<String>) {
  SpringApplication.run(ReadingListApplication::class.java, *args)
}

//override fun addViewControllers(registry: ViewControllerRegistry) {
//  registry.addRedirectViewController("/", "/readingList")
//}
